package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hcaptcha/react-hcaptcha",
-  "version": "1.11.3",
+  "version": "1.11.4",
```

Comment:
let's do 1.12.0 for this one
I'd suggest that we bump the version in the separate PR (to have it more visible in the commit history)